lighty-core/lighty-controller/src/main/java/io/lighty/core/controller/api/LightyServices.java

```diff
@@ -65,7 +63,7 @@
 
     ActorSystemProvider getActorSystemProvider();
 
-    SchemaContextProvider getSchemaContextProvider();
+    EffectiveModelContextProvider getEffectiveModelContextProvider();
```

Comment:
we should want to deprecate the old method (`getSchemaContextProvider()`) referencing newly created (`getEffectiveModelContextProvider`)

Reply:
SchemaContextProvider was completely removed from yangtools in Silicon. Should I create  deprecate getSchemaContextProvider() method with return value EffectiveModelContextProvider?

Reply:
OK, let's just switch those, 
use just `EffectiveModelContextProvider getEffectiveModelContextProvider()` as you did, as far as it is major release change in upstream